internal/testhelper/testhelper.go

```diff
@@ -0,0 +1,45 @@
+// Package testhelper implements a slam service definition with additional exported functions for
+// the purpose of testing
+package testhelper
+
+import (
+	"bufio"
+	"bytes"
+	"context"
+	"testing"
+
+	"github.com/golang/geo/r3"
+	"go.viam.com/rdk/components/camera"
+	"go.viam.com/rdk/pointcloud"
+	"go.viam.com/test"
+	"go.viam.com/utils/pexec"
+)
+
+// Service in the internal package includes additional exported functions relating to the data and
+// slam processes in the slam service. These functions are not exported to the user. This resolves
+// a circular import caused by the inject package.
+type Service interface {
+	StartDataProcess(cancelCtx context.Context, cam []camera.Camera, c chan int)
+	StartSLAMProcess(ctx context.Context) error
+	StopSLAMProcess() error
+	Close() error
+	GetSLAMProcessConfig() pexec.ProcessConfig
+	GetSLAMProcessBufferedLogReader() bufio.Reader
+}
+
+// TestComparePointCloudsFromPCDs is a helper function for checking GetPointCloudMapFull response along with associated pcd validity checks.
+func TestComparePointCloudsFromPCDs(t *testing.T, pcdInput, pcdOutput []byte) {
```

Comment:
Do we need this in viam-orb-slam3 (or maybe it will be removed in a future PR)?

Asking because it was not included in viam-cartographer: https://github.com/viamrobotics/viam-cartographer/pull/15#discussion_r1137547716

Reply:
Kat and I have been talking about this file a bit. We think that this entire file can be deleted in a future PR. I will delete this function for now though. Good catch!

Reply:
Ok, cool! Feel free to resolve.